app/assets/javascripts/autoform_form.js

```diff
@@ -19,14 +20,57 @@ $(document).ready(function() {
             return false;
         }
         else {
-            $('#autoform-error').hide();
-            $('#autoform-email').parent('.form-group').removeClass('has-error');
-            $('#autoform-domain').parent('.form-group').removeClass('has-error');
-            $('#autoform-form').hide();
-            $('#autoform-form-sent').show();
+            const formData = $(this).serialize();
+
+            $.post($(this).attr('action'), formData)
```

Comment:
skúšal som si to lokálne a pri správnom odoslaní sa to správne odošle a zapíše do formulára, ale vyskočí CORS policy error a na usera vyskočí hláška, že sa nepodarilo odoslať. Nejako sa nám to komplikuje s tým JS. Ale vracia to response 200, tak možno by bolo lepšie sa vrátiť k checkovaniu priamo status code, či je 200. A nie done a fail funkcie :/ Ak by si vedel vyriešiť tie cors problémy, tak super, ale ja som to doteraz vždy vzdal po dlhom boji, keď som na ne niekde narazil.